pymatviz/ptable.py

```diff
@@ -428,12 +429,8 @@ def get_elem_type_color(
         default: str = "white",
     ) -> str:
         """Get element type based color."""
-        try:
-            elem_type = df_ptable.loc[elem_symbol, "type"]
-            return self.elem_type_colors.get(elem_type, default)
-
-        except KeyError:
```

Comment:
This was intended to guard against a non-existent element (happy to discuss this if you think this is unnecessary), I would revert this and issue a warning in such case.

```python
    def test_get_elem_type_color(self) -> None:
        projector = PTableProjector(data=self.test_dict)

        assert projector.get_elem_type_color("H") == "green"
        assert projector.get_elem_type_color("Fe") == "blue"
        assert projector.get_elem_type_color("NONE", "Hello") == "Hello"
```

Reply:
I decide to remove this as we cannot expect the code to work for a non-existent element. This doesn't look necessary.